examples/ansible/test.fmf

```diff
@@ -0,0 +1,3 @@
+summary: Run a simple test written in ansible
+test: ansible-playbook -c local test.yml || exit 1
+require: ansible
```

Comment:
Could you add an explicit `framework: shell` ?

Reply:
Sure, added.